### ISSUE TITLE
Punnycode support

### ISSUE THREAD

**Issue by wtoorop** [NLnetLabs/ldns#25]
Copied from bugzilla

Viachaslau Khalikin 2018-08-09 17:01:33 CEST
At the moment drill interprets the cyrillic domain as escape sequence instead punycode. As a result DNS not give the right entry.

**Comment by wtoorop** [comment]
Copied from bugzilla

Glisha 2014-03-20 16:39:13 CET
When I query idn records without converting them to punnycode I get NXDOMAIN.

**Comment by sincomil** [comment]
Yes it is one of the stopping reasons from everyday use of drill in my case. Punnycode support is very important feature because of wide use of national TLDs
